app/registries/foreman/plugin.rb

```diff
@@ -37,10 +38,12 @@ class PluginRequirementError < Foreman::Exception; end
   class Plugin
     @registered_plugins = {}
     @tests_to_skip = {}
+    @importer_registry = Plugin::ImporterRegistry.new
     @report_scanner_registry = Plugin::ReportScannerRegistry.new
 
     class << self
       attr_reader   :registered_plugins
+      attr_accessor :tests_to_skip,:importer_registry, :report_scanner_registry
```

Comment:
The `attr_accessor` call below can go and there should be a space after the `,` :)